mypy/server/astdiff.py

```diff
@@ -239,6 +241,7 @@ def snapshot_definition(node: SymbolNode | None, common: SymbolSnapshot) -> Symb
             node.is_static,
             signature,
             is_trivial_body,
+            dataclass_transform_spec,
```

Comment:
This doesn't support equality and can't be used in snapshots. You can probably use the serialized form instead.